packages/accounts/src/index.ts

```diff
@@ -18,5 +18,7 @@ export {
 export type { KernelBaseValidatorParams } from "./kernel-zerodev/validator/base.js";
 
 //light-account exports
+export { LightAccountAbi } from "./light-account/abis/LightAccountAbi.js";
+export { LightAccountFactoryAbi } from "./light-account/abis/LightAccountFactoryAbi.js";
```

Comment:
we should definitely break this out of this PR

Reply:
k. will revert this change from this pr.